detections/endpoint/ssa___setting_credentials_via_powersploit_modules.yml

```diff
@@ -0,0 +1,45 @@
+name: Setting Credentials via PowerSploit modules
+id: 07b2a501-f967-4ddc-9f56-2dce46dfce44
+version: 1
+date: '2020-11-03'
+description: This detection identifies illegal setting of credentials via PowerSploit modules. 
+how_to_implement: You must be ingesting Windows Security logs from devices of interest, including the event ID 4688 with enabled command line logging.
+references: 
+- https://github.com/PowerShellMafia/PowerSploit
+type: SSA
+author: Stanislav Miskovic, Splunk
+search: '| from read_ssa_enriched_events()
+
```

Comment:
minor detail @miskoSplunk  whitespace on lines is for readability in this case for the SPLv2?

Reply:
Yes, not necessary, but I left blank lines so a reviewer can easily see the detection - which is extraction of fields that will be used and the where condition ... the rest is just a loading/reporting overhead of SSA.